docs/source/guides/speed.rst

```diff
@@ -0,0 +1,279 @@
+.. testsetup:: *
+
+    from pytorch_lightning.trainer.trainer import Trainer
+    from pytorch_lightning.callbacks.early_stopping import EarlyStopping
+    from pytorch_lightning.core.lightning import LightningModule
+
+.. _speed:
+
+#######################
+Speed up model training
```

Comment:
We should define this does this mean training time to convergence. If so how do we define convergence? Should Torch metrics get a shoutout.

Reply:
Not sure what you mean here, feel free to edit